internal/gps/internal/paths/paths.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package paths
+
+import "strings"
+
+// This was lovingly lifted from src/cmd/go/pkg.go in Go's code.
```

Comment:
This comment should describe what this function does, not where it was copied from.